dts/bindings/gpio/arducam,dvp-20pin-connector.yaml

```
@@ -0,0 +1,40 @@
+# Copyright 2025 tinyVision.ai Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  GPIO pins exposed on ArduCam 20-pin header camera connector,
+  originally designed to connect Arducam modules to Arduino Giga R1,
```

This makes sure there is a short "title" instead of the otherwise long sentence.
You may also used the new-ish `title` property, actually.

```suggestion
  GPIO pins exposed on ArduCam 20-pin header camera connector.
  Originally designed to connect Arducam modules to Arduino Giga R1,
```